seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/TestContainer.java

```diff
@@ -69,6 +74,11 @@ default Container.ExecResult restoreJob(String confFile, String jobId)
         throw new UnsupportedOperationException("Not implemented");
     }
 
+    default Container.ExecResult restoreJob(String confFile, String jobId, List<String> variables)
```

Comment:
```suggestion
    default Container.ExecResult restoreJob(String confFile, String jobId, String... variables)
```

Reply:
update this method

https://github.com/apache/seatunnel/blob/dev/seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/TestContainer.java#L67